data/json/materials.json

```diff
@@ -1551,6 +1551,24 @@
     "bash_dmg_verb": "damaged",
     "cut_dmg_verb": "damaged"
   },
+  {
+    "type": "material",
+    "id": "mercury",
+    "name": "Mercury",
+    "density": 13.53,
+    "specific_heat_liquid": 0.1395,
+    "specific_heat_solid": 0.1395,
```

Comment:
specific_heat_liquid and solid shouldn't be the same.

Reply:
I've only found this data on liquid mercury. Any suggestions what it could be like for solid mercury?

Reply:
−38.83 °C So whatever that converts to specific heat at?

Reply:
Unless I'm missing the point of how the game calculates temperature-related stuff, specific heat does depend on temperature and state of matter, but there seems to be no straightforward way to calculate specific heat at a different temperature. For instance, for water at 20 C the specific heat capacity is 4.187 kJ/kgK, for ice it's 2.108 kJ/kgK, and for steam it's 1.996 kJ/kgK. However, specific heat of solids is indeed lower than liquids, so I think I'll just set it to some kind of lower value.

Reply:
Hi guys, Mercury is a bit funny in this regard.
The specific heat capacity actually increases as you decrease temperature and get closer to the melting point. The specific heat capacity of solid mercury at melting point is 0,1419 kJ/kgK as published in Kozin, L. F., & Hansen, S. C. (2013). Mercury Handbook. Royal Society of Chemistry.
Decreasing the temperature further lowers the specific heat capacity as expected.

Reply:
Number updated, thanks for information.